specification/storageactions/resource-manager/Microsoft.StorageActions/stable/2023-01-01/storageactions.json

```diff
@@ -525,7 +525,10 @@
           "x-ms-client-flatten": true,
           "$ref": "#/definitions/StorageTaskProperties",
           "description": "Properties of the storage task."
-        }
+        },
+        "required": [
```

Comment:
The "required" is not one of the properties, it should be moved 1 level outside, or the swagger is not valid.